scripts/tuple_tree_generator/tuple_tree_generator/templates/struct.h.tpl

```diff
@@ -32,9 +32,13 @@ struct /*= struct | fullname =*/
   /**- for field in struct.fields **/
   /*= field.doc | docstring =*/
   /**- if field.const **/const /** endif -**/
+  private:
   /*= field | field_type =*/ The/*= field.name =*/ = /*= field | field_type =*/{};
   static_assert(Yamlizable</*= field | field_type =*/>);
 
+  public:
+  using /*= field.name =*/Type = /*= field | field_type =*/;
+
   const /*= field | field_type =*/ & /*= field.name =*/() const {
```

Comment:
Add another public here to group things nicely

Reply:
not sure i understand the rationale of this, currently it is being printed as
```
private:
 FieldType Field = {};
public:
 const FieltType& field() const ...
 FieltType& field()  ...
```

do you want another public inbetween the two?

Reply:
We use `public`/`private` to group things (see [B.16](https://pad.rev.ng/s/SJNUet_ku#B-CC)).
Group `using`s vs accessors with an additional `public`.

Reply:
ahh it is a using, i thought it was btween the two getters